src/widget/widget.cpp

```diff
@@ -1508,6 +1513,10 @@ void Widget::onGroupPeerAudioPlaying(int groupnumber, int peernumber)
     if (!g)
         return;
 
+    ToxId author = Core::getInstance()->getGroupPeerToxId(groupnumber, peernumber);
+    if(!author.isSelf())
+        updateFriendActivity(FriendList::findFriend(author));
+
```

Comment:
Actually, this is rather invalid.

In current groupchats it's doable, but in new groupchats it will not be possible, thus it's better to not introduce functionality that later on will be removed.

When we've talked about it yesterday, I've thought that groupchats are also being sorted `By Activity` when `Place groupchats at top of friend list` is not set. Since this is not the case, and groupchats aren't a subject to sorting `By Activity`, docs need to be corrected.

Also, I wonder how `FriendList::findFriend()` would handle PKs that aren't actually in the friendlist.